.circleci/config.yml

```diff
@@ -260,6 +260,66 @@ jobs:
             DOCKER_BUILD_ARGUMENTS="--cache-from securedrop-test-focal-py3:${fromtag:-latest}" securedrop/bin/dev-shell \
                   bash -c "pip3 install -U -q --upgrade pip && pip3 install -U -q --upgrade semgrep && make -C .. semgrep"
 
+  zap-vulnerability-scan:
+    machine:
+      image: ubuntu-2004:202010-01
+      enabled: true
+    environment:
+      DOCKER_API_VERSION: 1.23
+      BASE_OS: focal
+    parallelism: 3
```

Comment:
The three scan reports in (e.g.) <https://app.circleci.com/pipelines/github/freedomofpress/securedrop/5046/workflows/f1deaab2-98bc-40b5-9a55-105d13622adf/jobs/65622/artifacts> seem to be identical.  Is this `parallelism` necessary?